connectors/es/sink.py

```diff
@@ -164,7 +164,6 @@ async def _bulk_api_call():
                 for op, data in item.items():
                     if "error" in data:
                         self._logger.error(f"operation {op} failed, {data['error']}")
```

Comment:
I think we should count how often this happens, and at least log out the total at the end of the sync. The customer needs to know of 99% of their documents fail here, especially if we start calling such sync jobs "complete" instead of "error"